region.go

```diff
@@ -254,12 +252,3 @@ func AvailableSysTypes(region string) ([]string, error) {
 	}
 	return knownRegion.SysTypes, nil
 }
-
-// AllKnownSysTypes returns aggregated known system types from all regions.
-func AllKnownSysTypes() sets.Set[string] {
```

Comment:
but the question is - is this consumed anywhere right now?

Reply:
I seemed to be the only person interested in having one place where all this code would be. There was no action done on my working solution.

Reply:
yes, that's the reason we are removing the k8s.io deps so that we can import this code everywhere with no conflict with k8s deps

Reply:
@hamzy if this function is not consumed anywhere, will merge this code and find some other way of implementing this function withought k8s sets package